src/search/documents/hub.py

```diff
@@ -61,6 +61,17 @@ def prepare_name_suggest(self, instance):
             "weight": max(weight, 1),
         }
 
+    def prepare_name_suggest(self, instance):
```

Comment:
This shadows the previously defined method with the same signature (on line 51) making it effectively inaccessible. Was this intended?

Reply:
This is an oversight. The other method should be deleted!